bindings/iDynTree.i

```diff
@@ -302,8 +305,8 @@ TEMPLATE_WRAP_MOTION_FORCE(ForceVector3, WRAP_FORCE, SET_NAME_FOR_WRAPPER,,)
 
 %include "joints.i"
 
-%template(SolidshapesVector) std::vector<iDynTree::SolidShape*>;
-%template(linksSolidshapesVector) std::vector< std::vector<iDynTree::SolidShape *>>;
+%template(SolidShapesVector) std::vector<iDynTree::SolidShape*>;
+%template(linksSolidShapesVector) std::vector< std::vector<iDynTree::SolidShape *>>;
```

Comment:
As  I mentioned in another comment, it would be great to keep the `l` capital. 
```suggestion
%template(LinksSolidShapesVector) std::vector< std::vector<iDynTree::SolidShape *>>;
```

Reply:
I'm becoming dislexic XD, I read the middle ones to be capital as in the original variables. Sure I'll change that.

Reply:
Done @traversaro